src/views/surveys.vue

```diff
@@ -1,5 +1,12 @@
 <template>
 <div>
+  <Navbar @error="sendError" @editName="edit_name=true" v-bind:name="name" v-bind:utorid="utorid"></Navbar>
+
+  <v-dialog
+  v-model="edit_name">
+      <NameModal v-bind:user_package="{user_id:$route.query.user_id, name:name}" @close="edit_name=false" @error="sendError" @saveName="saveName"></NameModal>
```

Comment:
Make `user_package` a computed prop and don't access the global `$route` object. (Use `this.user_id`)